crates/starknet-devnet-core/src/starknet/starknet_config.rs

```diff
@@ -34,22 +37,67 @@ pub struct ForkConfig {
     pub block_number: Option<u64>,
 }
 
-#[derive(Clone, Debug)]
+pub fn serialize_fork_config<S>(fork_config: &ForkConfig, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
why not implement serialize on ForkConfig

Reply:
It took me some time to remember, but I wanted to make the `fork_config` part of the response to be similar to the `/fork_status` endpoint that is removed with this PR. So to have the whole `fork_config` be equal to `null` or `{}` in case the internal properties are undefined (url and block number).

Reply:
But now that I think of it, it's not that important to keep the compatibility. Changed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/433/commits/02fd772699eaec3a2ae2ddd3337dbcc78c2aeeca